src/test/java/hudson/plugins/jira/JiraCreateIssueNotifierTest.java

```diff
@@ -0,0 +1,176 @@
+package hudson.plugins.jira;
+
+import com.atlassian.jira.rest.client.api.domain.Component;
+import com.atlassian.jira.rest.client.api.domain.Issue;
+import com.atlassian.jira.rest.client.api.domain.Status;
+import hudson.EnvVars;
+import hudson.Launcher;
+import hudson.model.*;
```

Comment:
There are more than 5 imports and IDEA automatically converts it to wildcard. Any reason to avoid that in tests?
Just read http://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad and... there are mixed opinions..